pandas/core/indexes/range.py

```diff
@@ -544,6 +548,14 @@ def __floordiv__(self, other):
                                   fastpath=True)
         return self._int64index // other
 
+    def _reverse(self):
```

Comment:
This is already defined in terms of slicing, i.e. `RangeIndex(...)[::-1]` will return a `RangeIndex` with the opposite step and is performant.

Reply:
Aha, nice, thanks!